specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/examples/StorageAccountGetBlobInventoryPolicy.json

```diff
@@ -4,7 +4,7 @@
     "resourceGroupName": "res7687",
     "accountName": "sto9699",
     "blobInventoryPolicyName": "default",
-    "api-version": "2020-08-01-preview",
```

Comment:
Just to confirm, the blob inventory policy don't have "Version" any more.
In the original GA design, it should have version as "1.0".